content/en/serverless/distributed_tracing/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Distributed Tracing
+kind: documentation
+further_reading:
+    - link: 'tracing/serverless_functions'
+      tag: 'Documentation'	
```

Comment:
there is a trailing tab character after `'Documentation'` causing the link placehold_translations script to fail in build here: https://gitlab.ddbuild.io/DataDog/documentation/-/jobs/36622481#L2043

Reply:
Thank you so much for spotting!